wrangler-service/src/main/java/co/cask/wrangler/service/connections/ConnectionHandler.java

```diff
@@ -285,17 +291,18 @@ public void delete(HttpServiceRequest request, HttpServiceResponder responder,
    * @param id for which all the information is returned from the connection store.
    */
   @GET
-  @Path("connections/{id}")
+  @Path("contexts/{context}/connections/{id}")
   public void get(HttpServiceRequest request, HttpServiceResponder responder,
-                  @PathParam("id") String id) {
-    try {
-      Connection connection = store.get(id);
-      responder.sendJson(new ServiceResponse<>(ImmutableList.of(connection)));
-    } catch (ConnectionNotFoundException e) {
-      ServiceUtils.notFound(responder, e.getMessage());
-    } catch (Exception e) {
-      ServiceUtils.error(responder, e.getMessage());
-    }
+                  @PathParam("context") String namespace, @PathParam("id") String id) {
+    respond(request, responder, namespace, () -> new ServiceResponse<>(store.get(id)));
```

Comment:
Before the response was list of id. Should we keep it? `ServiceResponse(String message)` seems to be updating message not value.